src/components/Signup.tsx

```diff
@@ -0,0 +1,110 @@
+import { useMultiStepForm } from "./useMultiStepForm";
+import { FormEvent, useState } from "react";
+import styled from "styled-components";
+import { SignupEmailCard } from "./SignupEmailCard";
+import { SignupPasswordCard } from "./SignupPasswordCard";
+
+const SignupContainer = styled.div`
+  position: relative;
+  background: white;
+  border: 1px solid black;
```

Comment:
I guess this is a placeholder for the positioning. 
```suggestion
  border: 1px solid black; // remove from production code
```